test/test.hs

```diff
@@ -240,6 +411,13 @@ tests = testGroup "hevm"
     , testProperty "simpProp-equivalence-sym" $ \(LitProp p) -> ioProperty $ do
         let simplified = pand (Expr.simplifyProps [p])
         checkEquivProp simplified p
+    -- This would need to be a fuzz test I think. The SMT encoding of Keccak is not precise
+    -- enough for this to succeed
+    , ignoreTest $ testProperty "storage-slot-simp-property" $ \(StorageExp s) -> ioProperty $ do
```

Comment:
we need this I think: https://github.com/ethereum/hevm/issues/380  

The counterexamples I'm seeing here give two outputs that are very close together. I'm fine to merge as is and add this in a separate pr.